docs/development/threat_model.rst

```diff
@@ -44,10 +44,9 @@ Assumptions about the person installing SecureDrop, usually the admin
    to give the attacker system-level access that would be unreasonable.
 -  The person obtains an authentic copy of SecureDrop and its
    dependencies.
--  The person follows our guidelines for :doc:`deploying the system, setting
-   up the landing page for the
-   organization <../deployment_practices>`, and for :doc:`installing
-   SecureDrop <../install>`.
+-  The person follows our guidelines for deploying the system, setting
+   up the :ref:`Landing Page` for the
```

Comment:
it reads **setting up the Landing Page for** which is ok except for the case maybe ?

Reply:
![screen shot 2017-10-02 at 5 16 04 pm](https://user-images.githubusercontent.com/7832803/31104966-6abf1a40-a795-11e7-8713-6195383595c8.png)